### PR TITLE
Connector Listing

### LINKED ISSUE #23: Connector Directory Listing
Once the connector SDK has been developed, we need to be able to help other developers search and discover new connectors that have been created for conduit.

### PR DESCRIPTION
**Pull request by neovintage**
### Description

Describes philosophies and the connectors that we know exist in open source. Also will point developers to where to upvote or ask for new connectors.

Fixes #23

### CODE REVIEW

**Review by uchennakevinm1 — COMMENTED** [review]
Overall LGTM.  Repos for community connectors should be in https://github.com/orgs/conduitio-labs/repositories. Have instructed @maha-hajja @hariso to push DONE connectors to the aforementioned space.

File: README.md [hunk 1]
```diff
@@ -85,6 +85,8 @@ The Docker image includes the [UI](#ui), you can access it by navigating to `htt
 
 ## Connectors
 
+For the full list of available connectors, see the [Connector List](docs/connector-list.md). If there's a connector that you're looking for that isn't available in Conduit, please file an [issue](https://github.com/ConduitIO/conduit/issues/new?assignees=&labels=triage&template=3-connector-request.yml&title=Connector%3A+%3Cresource%3E+%5BSource%2FDestination%5D).
```
Comment (hariso) [inline README.md:88]: Nitpick: docs/connector-list.md -> docs/connectors.md

File: docs/connectors.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# Conduit Connectors
+
+Connectors are an integral part of Conduit. Conduit ships with a couple of connectors that are built into the service to help developers bootstrap pipelines much more quickly. The built-in connectors include Postgres, File, Kafka and Amazon S3 connectors.
+
+The difference between Conduit connectors and those you might find from other services is that Conduit connectors are Change Data Capture-first (CDC). CDC allows your pipeline to only get the changes that have happened over time instead of pulling down an entire upstream data store and then tracking diffs between some period of time. This is critical for building real-time event-driven pipelines and applications. But, we'll note where connectors don't have CDC capabilities.
+
+
+## Roadmap & Feedback
+
+Based on what the Conduit team hears from developers, a [roadmap]() has been built to facilitate the connectors that need to be built for Conduit. If you have a connector that you really want to see supported or lend your vote for a particular connector, check out that roadmap and upvote the associated issue!
```
Comment (hariso) [inline docs/connectors.md:10]: Nitpick: link for roadmap missing.

File: docs/connectors.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+At this time, Conduit does not have any commerically supported connectors.
+
+### Source & Destination
+
+Source means, does the connector have the ability to get data from the upstream data store. Destination means, does the connector have the ability to write to the downstream data store.
+
+### The List
+
+| Connector | Source | Destination | Support |
```
Comment (hariso) [inline docs/connectors.md:28]: Did we intentionally leave out the Kafka Connect wrapper from the list?

**Review by lovromazgon — COMMENTED** [review]
I didn't realize we have that many connectors already! It's about time we get started on those breaking changes that we have planned (OpenCDC) and then try to not introduce any new ones 😅

File: docs/connectors.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+# Conduit Connectors
+
+Connectors are an integral part of Conduit. Conduit ships with a couple of connectors that are built into the service to help developers bootstrap pipelines much more quickly. The built-in connectors include Postgres, File, Kafka and Amazon S3 connectors.
```
Comment (maha-hajja) [inline docs/connectors.md:3]: Here it mentions postgres as a builtin connector, but in the Readme it doesn't. It also doesn't mention the generator connector here.

File: docs/connectors.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+* `Conduit` - These are connectors that are built by the Conduit. Any issues or problems filed on those repos will be respond to by the Conduit team.
+* `Community` - A community connector is one where a developer created a connector and they're the ones supporting it not the Conduit team.
+* `Legacy` - Some connectors are built using non-preferred methods. For example, Kafka Connect connectors can be used on Conduit. This is considered a stop gap measure until `conduit` or `community` connectors are built.
+
+At this time, Conduit does not have any commerically supported connectors.
+
+### Source & Destination
+
+Source means, does the connector have the ability to get data from the upstream data store. Destination means, does the connector have the ability to write to the downstream data store.
```
Comment (maha-hajja) [inline docs/connectors.md:24]:
```suggestion
Source means that the connector has the ability to get data from the upstream data store. Destination means that the connector has the ability to write to the downstream data store.
```

File: docs/connectors.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+### The List
+
+| Connector | Source | Destination | Support |
+|-----------|-------|----|-------------|
```
Comment (maha-hajja) [inline docs/connectors.md:29]: IMO it would be better to push all the builtin connectors to the top of the list

File: docs/connectors.md (hunk 2 above)
Comment (neovintage) [reply docs/connectors.md:10]: fixed

File: docs/connectors.md (hunk 4 above)
Comment (neovintage) [reply docs/connectors.md:3]: fixed

File: docs/connectors.md (hunk 5 above)
Comment (neovintage) [reply docs/connectors.md:24]: fixed

File: docs/connectors.md (hunk 6 above)
Comment (neovintage) [reply docs/connectors.md:29]: I'm leaving the list alphabetical to make it easier to search.

File: docs/connectors.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
+# Conduit Connectors
+
+Connectors are an integral part of Conduit. Conduit ships with a couple of connectors that are built into the service to help developers bootstrap pipelines much more quickly. The built-in connectors include Postgres, File, Random Data Generator, Kafka and Amazon S3 connectors.
+
+The difference between Conduit connectors and those you might find from other services is that Conduit connectors are Change Data Capture-first (CDC). CDC allows your pipeline to only get the changes that have happened over time instead of pulling down an entire upstream data store and then tracking diffs between some period of time. This is critical for building real-time event-driven pipelines and applications. But, we'll note where connectors don't have CDC capabilities.
```
Comment (hariso) [inline docs/connectors.md:5]: > But, we'll note where connectors don't have CDC capabilities.

In the connector repos, not here, right?

File: docs/connectors.md (hunk 2 above)
Comment (neovintage) [reply docs/connectors.md:10]: argh.

File: docs/connectors.md (hunk 7 above)
Comment (neovintage) [reply docs/connectors.md:5]: For now, in the connector repos.

**Review by hariso — APPROVED** [review]

File: docs/connectors.md (hunk 2 above)
Comment (hariso) [reply docs/connectors.md:10]: I've approved, feel free to merge when you push this change too.

Comment (neovintage) [reply docs/connectors.md]: fixed.

**Pull request merged**